src/scripts/safari/Safari.ts

```diff
@@ -25,15 +27,18 @@ class Safari {
 
     public static load() {
         Safari.grid = [];
+        Safari.pokemonGrid = [];
         Safari.playerXY.x = 0;
         Safari.playerXY.y = 0;
         Safari.lastDirection = 'up';
         Safari.inBattle(false);
         Safari.inProgress(true);
         Safari.balls(this.calculateStartPokeballs());
         for ( let i = 0; i < this.sizeY(); i++) {
-            const row = [...Array(this.sizeX())].map(Number.prototype.valueOf, 0);
+            const row = [...Array(this.sizeX())].fill(0);
             Safari.grid.push(row);
+            const pokemonRow: Array<null> = [...Array(this.sizeX())].fill(null);
```

Comment:
What is the purpose of spreading here?
```suggestion
            const pokemonRow: Array<null> = Array(this.sizeX()).fill(null);
```

Reply:
Sorted 👍